README.md

```diff
@@ -54,9 +47,45 @@ async function handleEvent(event) {
 }
 ```
 
+### Options
+
+This function was designed to work with [Worker Sites](TODO:link-marketing-page), but you can customize the behavior of this function by passing the following options into the second argument to `getAssetFromKV`.
+
+#### `ASSET_NAMESPACE`
+
+type: KV Namespace Binding (required if not using Wrangler --site feature)
+
+We assume that your Worker includes a [binding to a Workers KV Namespace](TODO:link-documentation) named `__STATIC_CONTENT`. This binding can be configured by passing the `ASSET_NAMESPACE` option to `getAssetFromKV`. It is further assumed that this namespace consists of static assets such as html, css, javascript, or image files, keyed off of a relative path that roughly matches the assumed url pathname of the incoming request.
+
+#### `__STATIC_CONTENT_MANIFEST` (optional)
+
+type: text blob (JSON formatted)
+
+Workers Sites with Wrangler bundles up a text blob that maps request paths to content-hashed keys that are generated by Wrangler as a [cache-busting measure](TODO:link-sites-blog-post). If this option/binding is not present, the function will fallback to using the raw pathname to look up your asset in KV. If, for whatever reason, you have rolled your own implementation of this, you can include your own by passing a stringified JSON object where the keys are expected paths, and the values are the expected keys in your KV namespace.
+
+#### `mapRequestToAsset`
+
+type: function
+
+By default, mapRequestToAsset takes any path that ends in `/` or evaluates to an html file and appends `index.html` or `/index.html` for lookup in your Workers KV namespace. This works for most static site generators, but you can customize this behavior by passing your own function as `mapRequestToAsset`. The function should take a `Request` object as its only argument, and return a string representation of the path to be looked up in the asset manifest/KV.
+
+#### `cacheControl`
+
+type: object
+
+`cacheControl` allows you to configure options for both the Cloudflare Cache accessed by your Worker, and the browser cache headers sent along with your Workers' responses. The default values are as follows:
+
+```javascript
+let cacheControl = {
```

Comment:
If we are going to add this we should define each property

Reply:
e.g. 
```
##### bypassCache
type: number

Sets `Cache-Control` header `max-age` to the number passed in (in seconds) to the response which will be used by browser/client caches. X days by default.

Reply:
i mean, we have to add it, it's part of the API. so yah we have to add that

Reply:
added. lmk what you think.